crates/sound/src/stream.rs

```diff
@@ -4,9 +4,9 @@
 use std::{collections::VecDeque, sync::Arc};
 
 use thiserror::Error as ThisError;
-use vm_memory::{Le32, Le64};
+use vm_memory::{Address, Bytes, Le32, Le64};
 
-use crate::{virtio_sound::*, IOMessage, SUPPORTED_FORMATS, SUPPORTED_RATES};
+use crate::{virtio_sound::*, Error as StreamError, IOMessage, SUPPORTED_FORMATS, SUPPORTED_RATES};
```

Comment:
`StreamError` is a bit confusing here since this is like our generic error enum at the crate level.
I'd suggest something like `SoundError`.